lib/rucio/core/did_meta_plugins/filter_engine.py

```diff
@@ -17,10 +17,11 @@
 import operator
 from datetime import date, datetime, timedelta
 from importlib import import_module
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Callable, Iterable, Optional
```

Comment:
`Iterable` should be imported from `collections.abc` (see: https://docs.python.org/3/library/typing.html#typing.Iterable)

Reply:
Importing `Iterator`, and, as discussed in private messages, `Callable` from collections.abc now